src/app/game/snakeladder/Dice.tsx

```
@@ -0,0 +1,49 @@
+          >
+            {Array(num).fill(0).map((_, i) => (
+              <span key={i} className={styles.diceDot}></span>
+            ))}
+          </div>
+        ))}
+      </div>
+      <button 
+        className={styles.rollButton} 
+        onClick={handleClick}
+        disabled={disabled || rolling}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)